backend/src/packages/orders/order.controller.ts

```diff
@@ -0,0 +1,616 @@
+   *                - $ref: '#/components/schemas/Order/CreateOrderWithNotRegisteredUser'
+   *      security:
+   *        - {}
+   *        - bearerAuth: []
+   *      responses:
+   *        201:
+   *          description: Successful order creation.
+   *          content:
+   *            application/json:
+   *              schema:
+   *                $ref: '#/components/schemas/Order'
+   *        400:
+   *          description:
+   *            Order creation error
+   *          content:
+   *            application/json:
+   *              schema:
+   *                oneOf:
+   *                 - $ref: '#/components/schemas/DriverNotExistError'
+   *                 - $ref: '#/components/schemas/OrderCreationError'
+   *
+   */
+  private async create(
+    options: ApiHandlerOptions<{
+      body: OrderCreateRequestDto;
+      user: UserEntityObjectWithGroupT | null;
+    }>,
+  ): Promise<ApiHandlerResponse> {
+    const driver = await this.driverService.findById(options.body.driverId);
+
+    if (!driver) {
+      throw new NotFoundError({ message: HttpMessage.DRIVER_DOES_NOT_EXIST });
+    }
+
+    return {
+      status: HttpCode.OK,
+      payload: await this.orderService.create({
+        ...options.body,
+        userId: options.user?.id ?? null,
+        businessId: driver.businessId,
+      }),
+    };
+  }
+
+  /**
+   * @swagger
+   * /orders/{id}:
+   *    get:
+   *      tags:
+   *       - orders
+   *      summary: Get order by Id
+   *      description: Get order by Id
+   *      parameters:
+   *       - in: path
+   *         name: id
+   *         schema:
+   *           type: integer
+   *         required: true
+   *         description: Numeric ID of the order to find
+   *         example: 1
+   *      security:
+   *        - {}
+   *        - bearerAuth: []
+   *      responses:
+   *        200:
+   *          description: Order found
+   *          content:
+   *            application/json:
+   *              schema:
+   *                $ref: '#/components/schemas/Order'
+   *        404:
+   *          description:
+   *            Order with such ID does not found
+   *          content:
+   *            plain/text:
+   *              schema:
+   *                $ref: '#/components/schemas/OrderDoesNotExist'
+   *        401:
+   *          UnauthorizedError:
+   *            description:
+   *              You are not authorized
+   *          content:
+   *            plain/text:
+   *              schema:
+   *                $ref: '#/components/schemas/UnauthorizedError'
+   *
+   */
+  private async findOne(
+    options: ApiHandlerOptions<{
+      params: Id;
+      user: UserEntityObjectWithGroupT | null;
+    }>,
+  ): Promise<ApiHandlerResponse> {
+    let business;
+
+    if (options.user?.group.key === UserGroupKey.BUSINESS) {
+      business = await this.businessService.findByOwnerId(options.user.id);
+    }
```

Comment:
you don't need this block in controller layer. and you don't need to pass businessId, just pass user, and then in service inside this block  find business id and verify if order belongs to this business
  if (user?.group.key === UserGroupKey.BUSINESS) {}